src/npr/rules_dorado/06_align.smk

```diff
@@ -38,11 +35,19 @@ rule align:
         genome =  config['genome'].get(org, None),
         json = target_json,
         html = target_html,
+        do_align = config['info_dict']['do_align']
     threads:
-        10
+        10 # same number of threads will be applied to dorado-align and samtools
     shell:
         """
-        {dorado} aligner {params.genome} {input.bam_file} | samtools sort -@ {threads} -m 20G - > {output.file} 2>> {log}
+        echo "do_align: {params.do_align}" 2>> {log}
+        if [[ "{params.do_align}" == "do_align" ]]; then
+            {dorado} aligner -t {threads} {params.genome} {input.fq_file} | samtools sort -@ {threads} -m 20G - > {output.file} 2>> {log}
+        else
+            echo "No alignment needed, skip" 2>> {log}
+            mv {input.bam_file} {output.file}
```

Comment:
If no analysis is required, I'd guess more appropriately no analysis folder should be provided

Reply:
current output puts all files under transfer/Project/Data, there is no difference of data with analysis or not, maybe renaming the file is easier to indicate is BAM is aligned or not